code/modules/station_goals/dna_vault.dm

```diff
@@ -267,34 +275,49 @@ var/list/non_simple_animals = typecacheof(list(/mob/living/carbon/human/monkey,/
 		return
 	if(!completed)
 		return
-	if(!H.ignore_gene_stability)
-		to_chat(H, "<span class='notice'>[src] stabilizes your genes, granting you the ability to have multiple powers.</span>")
-		H.ignore_gene_stability = 1
+	var/datum/species/S = H.dna.species
+	if(NO_DNA in S.species_traits)
```

Comment:
This check is now necessary; as the DNA vault was previously genetics based, IPCs (and other potentially future species without DNA) weren't able to utilize it as setting blocks for them will fail; as this is all species datum based and altering vars more directly, we now have to explicitly check for this.

Reply:
From my own experience the DNA vault previously had no problems working with IPCs and other synthetics, the rework currently doesn't seem to work, when I tried to use it in a round earlier I got an error saying that I had no DNA.

Reply:
Actually speaking for a *lot* of previous experience, IPCs were very much able to use the DNA Vault before and now they cannot. I thought this change was intended.